Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -570,33 +570,55 @@ + (MAAdapterError *)toMaxError:(nullable NSError *)vungleError
             break;
         case 2: // Invalid AppID
         case 201: // Invalid PlacementID
+        case 207: // Invalid Placement Type
+        case 222: // Invalid Placement load type
         case 500: // BannerView: Invalid Size
             adapterError = MAAdapterError.invalidConfiguration;
             break;
+        case 119: // Json Encode Error
+            adapterError = MAAdapterError.internalError;
+            break;
+        case 202: // Ad already Consumed
+        case 203: // Ad is already loading
+        case 204: // Ad already loaded
+        case 205: // Ad is playing
+        case 206: // Ad already failed loading
+            adapterError = MAAdapterError.invalidLoadState;
+            break;
         case 210: // Ad Not Loaded
-            adapterError = MAAdapterError.noFill;
+            adapterError = isPlayFlow ? MAAdapterError.adNotReady : MAAdapterError.invalidLoadState;
+            break;
+        case 115: // Invalid IndexURL
+        case 302: // Invalid Ifa Status
+        case 305: // Mraid Bridge Error
+        case 400: // Concurrent Playback Unsupported
+            adapterError = MAAdapterError.adDisplayFailedError;
             break;
         case 212: // Placement Sleep
-            adapterError = MAAdapterError.invalidLoadState;
+            adapterError = MAAdapterError.noFill;
+            break;
+        case 217: // Ad response timeOut
+            adapterError = MAAdapterError.timeout;
             break;
+        case 220: // Server busy with retry after timer.
+        case 221: // Load ad during Server busy with retry after timer.
+            adapterError = MAAdapterError.serverError;
         case 304: // Ad Expired
+        case 307: // Ad Expired on play call.
             adapterError = MAAdapterError.adExpiredError;
             break;
-        case 303: // Ad Not Ready
-            adapterError = MAAdapterError.adNotReady;
-            break;
         case 600: // Native Asset Error
             adapterError = MAAdapterError.missingRequiredNativeAdAssets;
             break;
+        case 2000: // webView WebContent Process Did Terminate
+        case 2001: // webView Failed Navigation
+            adapterError = MAAdapterError.webViewError;
```

Comment:
minor - should this have a `break;` after assigning the error? Don't think this breaks anything, but it's more following the pattern.